src/databricks/labs/ucx/progress/directfs_access.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import replace
+
+from databricks.labs.lsql.backends import SqlBackend
+
+from databricks.labs.ucx.progress.history import ProgressEncoder
+from databricks.labs.ucx.progress.install import Historical
+from databricks.labs.ucx.source_code.base import DirectFsAccess
+from databricks.labs.ucx.source_code.directfs_access import DirectFsAccessOwnership
+
+
+class DirectFsAccessProgressEncoder(ProgressEncoder[DirectFsAccess]):
```

Comment:
shouldn't it belong to a notebook/sql query/dashboard, rather than being it's own top-level entity?